lightning/backend/backend.go

```diff
@@ -265,25 +260,39 @@ func (engine *OpenedEngine) Flush() error {
 
 // WriteRows writes a collection of encoded rows into the engine.
 func (engine *OpenedEngine) WriteRows(ctx context.Context, columnNames []string, rows Rows) error {
-	var err error
+	writer, err := engine.backend.LocalWriter(ctx, engine.uuid)
+	if err != nil {
+		return err
+	}
+	if err = writer.AppendRows(ctx, engine.tableName, columnNames, engine.ts, rows); err != nil {
+		return err
+	}
+	return writer.Close()
+}
```

Comment:
maybe just remove the whole method?

Reply:
There is still some tests using this method.

Reply:
if they are only used in tests, they can be converted to use the local writer.

(since i'm not so fond of apparently leaking the LocalWriter (not calling Close) when AppendRows returns an error, but it isn't worth "fixing" if no one actually uses it)

Reply:
Yes, I will fix it and add Close.